src/components/graphs/graph-helpers.js

```diff
@@ -80,28 +80,36 @@ function findMatchingMetadata(example, difference, meta) {
   return _.findWhere(meta, template);
 }
 
-function displayError(error, displayMethod) {
-  // Used to display any error (via `displayMethod`) generated in the
-  // process of showing a graph or table, so it handles networking
+export function errorMessage(error) {
```

Comment:
These changes seem familiar; does this branch need to be rebased?

Reply:
Rebasing